### PR TITLE
Get ready for the first OSS release

### PR DESCRIPTION
**Pull request by retohugi**
Hi guys,
Github suggested you two as reviewers - cool thing ;-)

so, here we go. You should be able to step by step read the commit messages to get the picture. Please do not merge, would like to get the build badge green first. 

Feedback very much appreciated!

\- reto

### CODE REVIEW

File: pushDocs.ps1 [hunk 1]
```diff
@@ -0,0 +1,37 @@
+param([string]$buildFolder, [string]$docSource, [string]$repoUrl, [string]$email, [string]$username, [string]$personalAccessToken)
```
Comment (yannisgu) [inline pushDocs.ps1]: personalAccessToken is not used

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      branch: master
+
+  - provider: NuGet
+    api_key: $(NuGetToken)
+    on:
+      appveyor_repo_tag: true
+
+deploy_script:
+- ps: >-
+    if($env:appveyor_repo_tag -eq 'true') {
```
Comment (yannisgu) [inline appveyor.yml]: maybe we should set `$ErrorActionPreference = 'Stop'` here

Comment (dawiddworak88) [reply appveyor.yml]: Yes, I'd opt for it too.

File: pushDocs.ps1 (hunk 1 above)
Comment (retohugi) [reply pushDocs.ps1]: nice catch. it's now part of the urlRepo.

**Pull request merged**